collect_app/proguard-rules.txt

```diff
@@ -5,6 +5,9 @@
 -dontwarn org.xmlpull.v1.**
 -dontwarn org.hamcrest.**
 -dontwarn com.rarepebble.**
+-dontwarn com.networknt.**
+-dontwarn android.content.res.**
+-dontwarn org.kxml2.io.**
```

Comment:
I haven't had to touch ProGuard for a while. Remind me what `dontwarn` is doing here?

Reply:
It ignores some weird errors I got like:
`com.android.tools.r8.utils.b: Library class android.content.res.XmlResourceParser implements program class org.xmlpull.v1`
It's enough for building and installing the app but we might run into some bugs on runtime too. We will see.